### PR TITLE
Point package.json to raw src code

### PR DESCRIPTION
**Pull request by ling1726**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Point package.json of N* packages to raw source code and create a prepublish step for N* modify package.json types and entrypoints to built code before publishing

#### Focus areas to test

(optional)

### CODE REVIEW

File: scripts/gulp/tasks/bundle.ts [hunk 1]
```diff
@@ -14,16 +14,18 @@ const packageName = config.package;
...
-    [
-      `${paths.packageDist(packageName)}/es/*`,
-      `${paths.packageDist(packageName)}/commonjs/*`,
-      `${paths.packageDist(packageName)}/dts`,
-    ],
+task('bundle:package:clean', () => {
+  const pathsToDelete = [
+    `${paths.packageDist(packageName)}/es/*`,
+    `${paths.packageDist(packageName)}/commonjs/*`,
+    `${paths.packageDist(packageName)}/dts`,
```
Comment (ecraig12345) [inline scripts/gulp/tasks/bundle.ts:21]: You can probably get rid of the windows fix below by doing this:
```suggestion
    paths.packageDist(packageName, 'es', '*'),
    paths.packageDist(packageName, 'es', '*'),
    paths.packageDist(packageName, 'es', '*'),
```

File: scripts/lernaAliasNorthstar.js [hunk 2]
```diff
@@ -1,28 +1,42 @@
 const lernaAlias = require('lerna-alias');
+const findGitRoot = require('./monorepo/findGitRoot');
+const path = require('path');
 
 // northstar packages should pull these from npm, not the repo
 const excludedPackages = ['@fluentui/date-time-utilities', '@fluentui/dom-utilities', '@fluentui/react-compose'];
 
+function deleteNorthstarAliases(aliases) {
```
Comment (ecraig12345) [inline scripts/lernaAliasNorthstar.js:8]: I'm confused, the whole point of this file was to be lerna-alias *specifically for* things under packages/fluentui.

Comment (ling1726) [reply scripts/lernaAliasNorthstar.js:8]: yes, I am trying to remove the aliases by pointing the entrypoints to raw src code, until publish time

Comment (ling1726) [reply scripts/lernaAliasNorthstar.js:8]: Since I haven't modified the entrypoints for packages outside of packages/fluentui I still need those aliases

Comment (ecraig12345) [reply scripts/lernaAliasNorthstar.js:8]: Ah okay, that makes sense. Can you add a comment about that?

Comment (ling1726) [reply scripts/lernaAliasNorthstar.js:8]: that's a good point ! I'll do that

File: scripts/tasks/ts.ts [hunk 3]
```diff
@@ -45,4 +47,7 @@ export const ts = {
     // Use default tsbuildinfo for this variant (since it's the only variant)
     return tscTask({ ...extraOptions, outDir: 'lib', module: 'commonjs' });
   },
+  build: () => {
+    return tscTask({ build: true });
+  },
```
Comment (ling1726) [inline scripts/tasks/ts.ts:52]: @ecraig12345 any thoughts on this ? I converted `react-codesandbox-northstar` to use tsc -b to be able to take a dep on the N* packages

**Comment by layershifter** [comment]
Closing for housekeeping 🏡 Hope that we will do it someday...

**Pull request closed without merging**